README.md

```diff
@@ -230,6 +233,8 @@ First the required values.
 
 Now the optional stuff.
 
+`firewallSgId` - set this to import/re-use an existing security group that will be applied to the load balancer and bastion host. See "Secruity" below.
```

Comment:
I think this is easy to infer from the examples, but just to verify: all values are of type `string`, right?

Reply:
most but not all! i went through the readme and added type indicators for all the config options.